readers/multishard.cc

```diff
@@ -803,8 +803,9 @@ future<> shard_reader_v2::do_fill_buffer() {
         remote_fill_buffer_result_v2 result;
     };
 
-    if (!_reader) {
-        fill_buf_fut = smp::submit_to(_shard, [this, gs = global_schema_ptr(_schema)] () -> future<reader_and_buffer_fill_result> {
```

Comment:
`fill_buf_fut` now seems to be unused, should be removed.

Reply:
yes